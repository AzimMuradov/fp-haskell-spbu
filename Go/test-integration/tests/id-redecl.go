// should fail with "panic: analyzer error: identifier b redeclared"

func main() {
  var a int;

  {
    var a int;
  };

  var b int;
  var b int;
}
