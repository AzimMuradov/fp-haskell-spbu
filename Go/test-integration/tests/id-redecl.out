panic: analysis error: identifier b redeclared
